tests/integration/pdo/test_prepared_stmt_1.inc

```diff
@@ -0,0 +1,12 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Nice idea using the `.inc`s for code reusability!  Could you please add comments to the files to convey the context they expect and the general gist of what they do?

Reply:
Is bb4d7fba645f103122382e7a4943e4bb39f11797 what you had in mind?